main/src/hadd.cxx

```diff
@@ -286,6 +288,30 @@ int main( int argc, char **argv )
             }
          }
          ++ffirst;
+      } else if (strcmp(argv[a], "-objects") == 0) {
+         if (a + 1 != argc && argv[a + 1][0] != '-') {
+            fileObjects = argv[a + 1];
+            ++a;
+            ++ffirst;
+         } else {
+            std::cerr << "-objects: no objects specified. Parameter ignored.\n";
+         }
+         ++ffirst;
+      } else if (strcmp(argv[a], "-objectmergetype") == 0) {
+         if (a + 1 != argc && argv[a + 1][0] != '-') {
+            TString tmp = argv[a + 1];
+            if (tmp == "SkipListed")
```

Comment:
hadd should give an error if you try specifying the option twice. However I suggest you first rebase your PR on master because the argument parsing code has changed significantly and you should integrate your change with it.